mobile/android/app/build.gradle

```diff
@@ -138,7 +138,7 @@ android {
         applicationId "com.rightonnew"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 6
+        versionCode 8
```

Comment:
This is the correct build number (and version 8 has been built through Android Studio). There was some last minute builds trying to get better crash reporting through bugsnap to understand the crash more but ultimately it was simpler to just swap out the unneeded and problematic `TextInput`